lib/_http_incoming.js

```diff
@@ -47,7 +47,7 @@ function IncomingMessage(socket) {
   // response (client) only
   this.statusCode = null;
   this.statusMessage = null;
-  this._client = socket; // deprecated
+  this.client = socket;
```

Comment:
A quick question: why suggest using «socket or connection» instead of just one of those? «Use socket instead» would be more straightforward.

`.connection` is from the first verion of the http module, `.socket` got there when the http module was replaced with the second version. Both of those go as far as node 0.1.90 (node 0.2.0). Then '.connection' got back in for compatibility.

There is no actual need for keeping both of them, except for backwards compatibility. If one of those would be deprecated in further versions, the current message would add more inconvenience to the migration process:
— «client is deprecated, use socket or connection instead»
— ok, connection
— «connection is deprecated¸ use socket instead»
— …

Why not choose one of them now and recommend it? `socket` is used internally atm.

I do not agitate for immediate `.connection` deprecation, I just wanted to say that altering the message a bit would make further deprecation of one of those in some later version (if that happens) more convenient. Also it would not raise an extra question «should I use `socket` or `connection` here instead of `client`?».